open-company/open-company-web
# Update sections picker

Card: https://trello.com/c/D3ZJwqpQ

To test:
- open Cmail
- pick a section
- switch to fullscreen (or to collapsed mode if you are already)
- change section again
- [ ] check the sections picker reflect the new style (from the Trello card)
